src/components/ConfigureDateTimeFormatModal.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Configure date time format") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content class="ion-padding">
+    
+    <ion-item>
+      <ion-label color="medium">{{ $t("CSV input") }}</ion-label>
+      <ion-label slot="end">{{ arrivalDate }}</ion-label>
+    </ion-item> 
+    <ion-item>
+      <ion-label color="medium">{{ $t("Expected input") }}</ion-label>
+      <ion-label slot="end">{{ currentDate }}</ion-label>
+    </ion-item>
+    <ion-item lines="full">
+      <ion-label>{{ $t("Custom date format") }}</ion-label>
+      <ion-input clear-input='true' @keyup.enter="dateTimeFormat = $event.target.value;" v-model="dateTimeFormat" :value="dateTimeFormat" :placeholder="defaultDateTimeFormat" />
+    </ion-item>
+
+    <ion-item class="ion-padding-bottom" lines="none">
+      <a href="https://moment.github.io/luxon/#/formatting?id=table-of-tokens" target="_blank">{{ $t('View all date time formats supported by the HotWax Import app.') }}</a>
+    </ion-item>
+          
+    <ion-item class="ion-padding-bottom">
```

Comment:
Use class ion-margin-top instead of ion-padding-bottom

Reply:
Updated